### PR TITLE
[doc] advice when targeting both android/ios

### PR DESCRIPTION
**Pull request by lklots**
Based on a discussion with silhouettes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ Once you've followed the general-purpose ["getting started"](http://codepush.too
 npm install --save react-native-code-push@latest
 ```
 
-As with all other React Native plugins, the integration experience is different for iOS and Android, so perform the following setup steps depending on which platform(s) you are targetting.
+As with all other React Native plugins, the integration experience is different for iOS and Android, so perform the following setup steps depending on which platform(s) you are targeting. Note, if you are targeting both platforms it is recommended to create separate code push applications for each platform.
```
Comment (silhouettes) [inline README.md]: Could you change `code push` to `CodePush`?

**Comment by vojtechnovak** [comment]
Hello! I'd be interested in knowing why exactly this is recommended.? Is it only to control whether an update that affects say only android will be downloaded by iOS devices too?

**Comment by silhouettes** [comment]
@vojtechnovak - yes - currently, we don't have any logic to distinguish between the two platforms, so it's important that they are kept separate.

File: README.md (hunk 1 above)
Comment (silhouettes) [inline README.md]: I will merge this and then fix it up afterwards. Thanks for the PR!

**Pull request merged**